src/ort_genai_c.cpp

```diff
@@ -292,15 +292,15 @@ OgaResult* OGA_API_CALL OgaGenerator_AppendTokenSequences(OgaGenerator* oga_gene
   }
 
   auto input_ids = Generators::PadInputs(span_sequences, generator.model_->config_->model.pad_token_id);
-  generator.AppendTokens(input_ids);
+  generator.AppendTokens(Generators::cpu_span<const int32_t>(input_ids.data(), input_ids.size()));
```

Comment:
do we need an explicit conversion from `std::vector<int32_t>` to `cpu_span<const int32_t>`? might be nicer if it was automatic.